CHANGELOG.md

```diff
@@ -1,7 +1,23 @@
 # Parse-SDK-JS
 
 ### master
-[Full Changelog](https://github.com/parse-community/Parse-SDK-JS/compare/2.18.0...master)
+[Full Changelog](https://github.com/parse-community/Parse-SDK-JS/compare/2.19.0...master)
+
+## 2.19.0
+[Full Changelog](https://github.com/parse-community/Parse-SDK-JS/compare/2.18.0...2.19.0)
+
+**Features**
+- New error code 210 (MFA_ERROR) ([#1268](https://github.com/parse-community/Parse-SDK-JS/pull/1268))
```

Comment:
Not sure if these new error codes are required anymore, since https://github.com/parse-community/parse-server/pull/6977 has been closed in favor of a general interface approach in https://github.com/parse-community/parse-server/pull/7052. @dblythy, @Moumouls what do you think?

Reply:
You can remoce this error code yes, i think it's not needed any more due to the futur system

Reply:
The MFA adapter will still need to throw a "special" error if a login is attempted without a TOTP token. Maybe this should be changed to a more generic error such as "210 (AUTH_ERROR)"

Reply:
Okay, I guess it's unclear at this point what the new error codes should be. Let's just keep in mind to make a decision on these codes so that they don't remain there unnecessarily. @Moumouls I added that to a task of your auth adapter PR, just to be sure.

Reply:
Yes, I hope I will be able to finish my PR today to remove the draft label and then start the review process with the team :)